ci-operator/config/redhat-developer/helm/redhat-developer-helm-release-3.11__helm-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,57 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+binary_build_commands: make -f openshift-ci/Makefile build
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tests
+images:
+- dockerfile_path: openshift-ci/Dockerfile.tests
+  from: base
+  to: helm
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tests:
+- as: helm-interop-aws
+  cron: 0 6 * * 1
+  remote_api: true
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "jira_epic":"LPTOCPCI-290"},
+          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "jira_epic":"LPTOCPCI-290"},
+          {"step": "helm-acceptance-test", "failure_type": "pod_failure", "classification": "Test Execution", "jira_project": "LPTOCPCI", "jira_epic":"LPTOCPCI-290"},
+          {"step": "helm-acceptance-test", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "HELM", "jira_epic":"LPTOCPCI-290"}
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
+      FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+    test:
+    - as: helm-acceptance-test
+      commands: |
+        make -f openshift-ci/Makefile build test-acceptance
+        cp -r acceptance-testing/.acceptance/*.xml ${ARTIFACT_DIR}/
```

Comment:
Make sure xml files should match the syntax `junit_*.xml` in `${ARTIFACT_DIR}`. Testgrid uses this syntax to report the test results.